### ISSUE TITLE
v9.x CORS does't work via webPreferences webSecurity=false

### ISSUE THREAD

**Issue by maskerliu** [electron/electron#24217]
### Preflight Checklist

* [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details

* **Electron Version:**
  * 9.0.0 and later
* **Operating System:**
  * macOS 10.13.6
* **Last Known Working Electron version:**
  * 8.3.3

### Expected Behavior
   in BrowserWindow set to do http CORS request 
    webPreferences: {
      webSecurity: false
    },
   in low version electron, can do CORS request.

### Actual Behavior
Access to XMLHttpRequest at 'https://xxx' from origin 'http://localhost:9080' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.

**Comment by vladimiry** [comment]
#23664

**Comment by HashimotoYT** [comment]
Duplicate of #23664
